spec/models/patient_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Patient do
+  describe "relationships" do
+    it { should have_many :doctor_patients }
+    it { should have_many(:doctors).through(:doctor_patients) }
+  end
+  
+  describe "validations" do
+    it { should validate_presence_of :name }
+    it { should validate_presence_of :age }
+    it { should validate_numericality_of(:age).only_integer }
+  end
+
+  before(:each) do 
+    @hospital_1 = Hospital.create(name: "Mercy Hospital")
+    @hospital_2 = Hospital.create(name: "Jersey Shore Medical Center")
+
+    @doctor_1 = Doctor.create!(hospital_id: @hospital_1.id, name: "Dr. House", specialty: "Cancer Treatment", university: "Princeton")
+    @doctor_2 = Doctor.create!(hospital_id: @hospital_1.id, name: "Dr. Grey", specialty: "Spinal", university: "Notre Dame")
+    
+    @doctor_3 = Doctor.create!(hospital_id: @hospital_2.id, name: "Dr. Love", specialty: "Emotional Sciences", university: "Harvard")
+
+    @patient_1 = Patient.create(name: "Fred", age: 19)
+    @patient_2 = Patient.create(name: "Shaggy", age: 17)
+    @patient_3 = Patient.create(name: "Old Man Jenkins", age: 87)
+    @patient_4 = Patient.create(name: "Daphne", age: 18)
+
+    @doctor_patient_1 = DoctorPatient.create!(doctor: @doctor_1, patient: @patient_1)
+    @doctor_patient_2 = DoctorPatient.create!(doctor: @doctor_1, patient: @patient_2)
+    @doctor_patient_3 = DoctorPatient.create!(doctor: @doctor_1, patient: @patient_3)
+
+    @doctor_patient_4 = DoctorPatient.create!(doctor: @doctor_2, patient: @patient_1)
+  end
+
+  describe "instance methods" do
+    it "self.alphabetical_adults" do
+      all_patients = Patient.all
+      alphabetical_adults_array = [@patient_1, @patient_3]
+
+      expect(all_patients.alphabetical_adults).to eq(alphabetical_adults_array)
```

Comment:
Since this is a class method, you don't need line 38. Your assertion can just be: `expect(Patient.alphabetical_adults).to eq(alphabetical_adults_array)`